server/src/resolvers/StructureDefinition.ts

```diff
@@ -61,12 +62,23 @@ export const StructureDefinition = objectType({
       },
     })
 
-    t.field('attributes', {
-      type: 'JSON',
+    t.list.field('attributes', {
+      type: 'AttributeDefinition',
       description: 'Structured version of the attributes',
       resolve: (parent: any) => parent.attributes,
     })
 
+    t.list.field('extensions', {
+      type: 'StructureDefinition',
+      description: 'List of extensions on this type',
+      resolve: async (parent: any, { elementId }) => {
```

Comment:
Sorry, I don't get where this `elementId` comes from

Reply:
yeah, we don't need that anymore, sorry I forgot